deluge/core/core.py

```diff
@@ -359,8 +359,8 @@ def do_remove_torrents():
             for torrent_id in torrent_ids:
                 try:
                     self.torrentmanager.remove(torrent_id, remove_data=remove_data, save_state=False)
-                except InvalidTorrentError as ex:
-                    errors.append((torrent_id, ex))
+                except InvalidTorrentError:
+                    errors.append(torrent_id)
```

Comment:
Updated with a fixup that reverts the change and uses a string value instead of the exception. What do you think?